config/i18n.json

```diff
@@ -928,7 +928,7 @@
     "ShowNewItems": "Show a red dot on new items",
     "ExpandSingleCharacter": "Show all characters",
     "SingleCharacter": "Only show current character",
```

Comment:
```suggestion
    "SingleCharacter": "Single-Character Mode",
```
i'd be fine with promoting "Single-Character Mode" to the actual name of the feature.
the current separateness and colon structure don't really make sense.

Reply:
I could go either way. Single character mode maybe could be interpreted in some weird way.

Reply:
Single-Character View?

Reply:
Yeah that's clearer